### PR TITLE
feat: add `LndClient.send_payment_sync`

### PR DESCRIPTION
**Pull request by ramontayag**
Call the [SendPaymentSync](https://api.lightning.community/#sendpaymentsync) gRPC endpoint.

### CODE REVIEW

File: lib/lnd_client.ex [hunk 1]
```diff
@@ -118,6 +119,19 @@ defmodule LndClient do
     GenServer.call(__MODULE__, { :add_invoice, invoice })
   end
 
+  @doc """
+  Takes a %Lnrpc.SendRequest and attempts to pay the invoice
+
+  ## Examples
+
+      iex> Lnrpc.SendRequest.new(payment_request: "lnbcrt500paymentrequest") |> LndClient.send_request_sync
```
Comment (RooSoft) [inline lib/lnd_client.ex]: Seems like `LndClient.send_request_sync` should be `LndClient.send_payment_sync` here

**Review by RooSoft — CHANGES_REQUESTED** [review]
Minor documentation change

**Comment by RooSoft** [comment]
Been testing the code, it works, will merge after the requested change has been applied

File: lib/lnd_client.ex (hunk 1 above)
Comment (ramontayag) [reply lib/lnd_client.ex]: Ah yes. Good catch!

**Comment by RooSoft** [comment]
Thanks @ramontayag, keep the PRs coming!

**Pull request merged**